apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QCqlIT.java

```diff
@@ -0,0 +1,39 @@
+package io.stargate.sgv2.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.common.testresource.StargateTestResource;
+import org.junit.jupiter.api.Test;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(StargateTestResource.class)
+public class RestApiV2QCqlIT extends RestApiV2QIntegrationTestBase {
```

Comment:
Missing test cases:

* CQL not valid, should end up in `400`
* Body empty, validation kicks in